ignite/cmd/generate.go

```diff
@@ -33,9 +33,9 @@ meant to be edited by hand.
 	flagSetClearCache(c)
 
 	c.AddCommand(NewGenerateGo())
-	c.AddCommand(NewGenerateTSClient())
-	c.AddCommand(NewGenerateComposables())
-	c.AddCommand(NewGenerateHooks())
+	// c.AddCommand(NewGenerateTSClient()) // disabled until ts-client works better, in the meantime, ignite app web is the way to go
```

Comment:
What do you think about using the hide option only from the command?

Reply:
If we just want to hide them and no disable them then sure